src/quo/components/list_items/network_list_item/view.cljs

```diff
@@ -0,0 +1,57 @@
+(ns quo.components.list-items.network-list-item.view
+  (:require
+    [quo.components.list-items.network-list-item.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.foundations.resources :as quo.resources]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]))
+
+(defn- info
+  [{:keys [network-name label]}]
+  [rn/view {:style style/info}
+   [rn/image
+    {:source (quo.resources/get-network network-name)
```

Comment:
perhaps it's better we pass the source in directly? 
cc @ulisesmac, @briansztamfater 🤔

Reply:
The benefit of doing it in the component is that we don't have to require `quo.resources` in every file we use the `network-list` component.

Reply:
any preference here @ulisesmac @briansztamfater ?

Reply:
Hey guys, sorry for the late response, I completely missed this notification.

@J-Son89 we could refactor these network images in the same way we did it for the token component, wdyt? (of course, in a later PR).

@Pau1fitz I'd say it's better to follow the common approach for the sake of consistency.
Alos, if we refactor these calls it's easier if most of the code (ideally all) follows a single approach.

Reply:
have updated to pass in the image